eng/release/Scripts/PublishToBlobAccounts.ps1

```diff
@@ -0,0 +1,121 @@
+[CmdletBinding(SupportsShouldProcess)]
+Param(
+    [Parameter(Mandatory=$true)][string]$AzCopyPath,
+    [Parameter(Mandatory=$true)][string]$BuildNumber,
+    [Parameter(Mandatory=$true)][string]$ReleaseVersion,
+    [Parameter(Mandatory=$true)][string]$DotnetStageAccountKey,
+    [Parameter(Mandatory=$true)][string]$DestinationAccountName,
+    [Parameter(Mandatory=$true)][string]$DestinationAccountKey,
+    [Parameter(Mandatory=$true)][string]$ChecksumsAccountName,
+    [Parameter(Mandatory=$true)][string]$ChecksumsAccountKey
+)
+
+$ErrorActionPreference = 'Stop'
+Set-StrictMode -Version 2.0
+
+$sourceAccountName = 'dotnetstage'
+$sourceContainerName = 'dotnet-monitor'
+$destinationContainerName = 'dotnet'
+
+function Generate-Source-Uri{
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$true)][string]$AssetType
+    )
+
+    return "https://$sourceAccountName.blob.core.windows.net/$sourceContainerName/$BuildNumber/${AssetType}Assets/*"
+}
+
+function Generate-Destination-Uri{
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$true)][string]$AccountName
+    )
+
+    return "https://$AccountName.blob.core.windows.net/$destinationContainerName/diagnostics/monitor/$ReleaseVersion"
+}
+
+function Generate-Sas-Token{
+    [CmdletBinding()]
+    Param(
+        [Parameter(Mandatory=$true)][string]$StorageAccountName,
+        [Parameter(Mandatory=$true)][string]$ContainerName,
+        [Parameter(Mandatory=$true)][string]$AccountKey,
+        [Parameter(Mandatory=$true)][string]$Permissions
+    )
+
+    $context = New-AzStorageContext -StorageAccountName $StorageAccountName -StorageAccountKey $AccountKey;
+    return New-AzStorageContainerSASToken -Container $ContainerName -Context $context -Permission $Permissions -ExpiryTime (Get-Date).AddHours(1.0);
```

Comment:
Note that it's possible that if StartTime defaults to the current time and the transfer operation is attempted very quickly, the server may reject it due to clock skew. Consider setting an explicit StartTime.